RealmSwift/LinkingObjects.swift

```diff
@@ -274,7 +274,8 @@ public final class LinkingObjects<T: Object>: LinkingObjectsBase {
        is empty.
      */
     public func minimumValue<U: MinMaxType>(ofProperty property: String) -> U? {
-        return rlmResults.min(ofProperty: property) as! U?
+        guard let value = rlmResults.min(ofProperty: property) else { return nil }
```

Comment:
Probably a matter of preference, but rather than all the guard's, else's and dual returns, this pattern could be replaced by `map` throughout:

```swift
return rlmResults.min(ofProperty: property).map(U.bridging)
```